apps/admin-portal/src/components/roles/create-role-wizard/role-wizard.tsx

```diff
@@ -101,89 +101,16 @@ export const CreateRoleWizard: FunctionComponent<CreateRoleProps> = (props: Crea
         setPartiallyCompletedStep(undefined);
     }, [ partiallyCompletedStep ]);
 
-    /**
-     * Method to handle the create role wizard finish action.
-     * 
-     */
-    const handleRoleWizardFinish = () => {
-        addRole(wizardState)
-    };
-
-    /**
-     * Generates a summary of the wizard.
-     *
-     * @return {any}
-     */
-    const generateWizardSummary = () => {
-        if (!wizardState) {
-            return;
-        }
-
-        return wizardState;
-    };
-
-    // Create role wizard steps
-    const WIZARD_STEPS = [{
-        content: (
-            <RoleBasics
-                triggerSubmit={ submitGeneralSettings }
-                onSubmit={ (values) => handleWizardSubmit(values, WizardStepsFormTypes.BASIC_DETAILS) }
-            />
-        ),
-        icon: ApplicationWizardStepIcons.general,
-        title: "Basic Details"
-    },{
-        content: (
-            <PermissionList
-                triggerSubmit={ submitPermissionList }
-                onSubmit={ (values) => handleWizardSubmit(values, WizardStepsFormTypes.PERM_LIST) }
-            />
-        ),
-        icon: ApplicationWizardStepIcons.protocolConfig,
-        title: "Permission Selection"
-    },{
-        content: (
-            <AddRoleUsers
-                triggerSubmit={ submitRoleUserList }
-                onSubmit={ (values) => handleWizardSubmit(values, WizardStepsFormTypes.USER_LIST) }
-            />
-        ),
-        icon: ApplicationWizardStepIcons.protocolSelection,
-        title: "Assign user roles"
-    },{
-        content: (
-            <CreateRoleSummary
-                triggerSubmit={ finishSubmit }
-                onSubmit={ handleRoleWizardFinish }
-                summary={ generateWizardSummary() }
-            />
-        ),
-        icon: ApplicationWizardStepIcons.summary,
-        title: "Summary"
-    }]
-    
-    /**
-     * Handles wizard step submit.
-     *
-     * @param values - Forms values to be stored in state.
-     * @param {WizardStepsFormTypes} formType - Type of the form.
-     */
-    const handleWizardSubmit = (values: any, formType: WizardStepsFormTypes) => {
-        setCurrentWizardStep(currentStep + 1);
-        setWizardState(_.merge(wizardState, { [ formType ]: values }));
-    };
-
     /**
      * Method to handle create role action when create role wizard finish action is triggered.
      * 
      * @param basicData - basic data required to create role.
      * @param permissions - permissions selected for the created role.
      */
     const addRole = (basicData: any): void => {
-        let members: CreateRoleMemberInterface[] = [];
-
-        if (basicData.RoleUserList.length > 0) {
-            basicData.RoleUserList.forEach(user => {
+        const members: CreateRoleMemberInterface[] = [];
+        if (basicData.RoleUserList.users.length > 0) {
+            basicData.RoleUserList.users.forEach(user => {
```

Comment:
Could assign `basicData.RoleUserList.users` to a `const` and use

Reply:
fixed with commit f0e96ae